controllers/nnf_workflow_controller_helpers.go

```diff
@@ -1042,27 +1055,18 @@ func (r *NnfWorkflowReconciler) createContainerJobs(ctx context.Context, workflo
 	labels[nnfv1alpha1.PinnedContainerProfileLabelName] = profile.GetName()
 	labels[nnfv1alpha1.PinnedContainerProfileLabelNameSpace] = profile.GetNamespace()
 	labels[nnfv1alpha1.DirectiveIndexLabel] = strconv.Itoa(index)
+	labels[nnfv1alpha1.DirectiveIndexLabel] = strconv.Itoa(index)
```

Comment:
Are you setting this twice on purpose?

Reply:
No, I think this was a merge artifact from when I rebased. Good catch.